slack_bolt/context/fail/async_fail.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional
+
+from slack_sdk.web.async_client import AsyncWebClient
+from slack_sdk.web.async_slack_response import AsyncSlackResponse
+
+
+class AsyncFail:
+    client: AsyncWebClient
+    function_execution_id: Optional[str]
+
+    def __init__(
+        self,
+        client: AsyncWebClient,
+        function_execution_id: Optional[str],
+    ):
+        self.client = client
+        self.function_execution_id = function_execution_id
+
+    async def __call__(self, error: str = "") -> AsyncSlackResponse:
```

Comment:
Do we want to allow no error here? I believe we don't need to have the default value here.

Reply:
I agree we don't need the default value here 👍